src/spec/template_spec.js

```diff
@@ -335,4 +335,35 @@ describe('Template Page', () => {
     // Assert
     expect(templatePage.getTotalInvalidFields()).toBe(expectedTotalInvalidFields);
   });
+
+  it('should not show the mseditor button when the token forceMseditor is null', () => {
+    // Arrange
+    beginAuthenticatedSession();
+    var templatePage = new TemplatePage();
+
+    // Act
+    browser.get('/#/templates/new');
+
+    // Assert
+    expect(templatePage.isTemplateHtmlRawInputDisplayed()).toBe(true);
+    expect(templatePage.isTemplateMsEditorInputDisplayed()).toBe(false);
+  });
+
+  it('should show the mseditor button when the token forceMseditor is not null', () => {
```

Comment:
I think that `forceMseditor is true` or `forceMseditor is truly` is better than `forceMseditor is not null`